pkg/controller/iperf/iperf_resources.go

```diff
@@ -60,6 +62,65 @@ func generateServerPod(namespacedName types.NamespacedName, nodeSelectorValue st
 
 }
 
+func generateTestServerPod(namespacedName types.NamespacedName, nodeSelectorValue string) *corev1.Pod {
+	return &corev1.Pod{
+		TypeMeta: metav1.TypeMeta{
+			Kind:       "Pod",
+			APIVersion: "v1",
+		},
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      namespacedName.Name,
+			Namespace: namespacedName.Namespace,
+			Labels: map[string]string{
+				"app": "testserver",
+			},
+		},
+		Spec: corev1.PodSpec{
+			Containers: []corev1.Container{
+				{
+					Name:  "testserver",
+					Image: "gcr.io/google_containers/echoserver:1.4",
+					Ports: []corev1.ContainerPort{
+						{
+							ContainerPort: 8080,
+							Protocol:      corev1.ProtocolTCP,
+						},
+					},
+					VolumeMounts: []corev1.VolumeMount{
+						{
+							Name:      "var-lib-nginx",
+							MountPath: "/var/lib/nginx.",
```

Comment:
I think there is a typo here.

```suggestion
							MountPath: "/var/lib/nginx",
```

Reply:
fixed. Not sure how that dot snuck into the commit. I swear I tested this before I pushed :)